restapi/src/main/java/io/stargate/web/resources/Converters.java

```diff
@@ -78,336 +82,762 @@
     final Map<String, Object> map = new HashMap<>(defs.size());
     for (final Column column : defs) {
 
-      map.put(column.name(), transformObjectToJavaObject(row.getObject(column.name())));
+      map.put(column.name(), toJsonValue(row.getObject(column.name())));
```

Comment:
Being paranoid since this is also called in v1, nothing changes about the response. Right?

Reply:
It doesn't impact the structure of the response, but changes how some of the column values are represented. I can keep v1 exactly the way it was, but that includes the bugs about large numbers, map keys, and severely broken UDTs and tuples. For example:

* before:
    ```
    {"count":1,"data":[{"a":{"type":{"keyspace":{},"name":{},"frozen":false,"fieldNames":[{}],"fieldType
    s":[{"protocolCode":9,"detached":false}],"attachmentPoint":{"protocolVersion":"V4","codecRegistry":{
    }},"detached":false,"protocolCode":48},"formattedContents":"{i:2}"},"k":1}]}
    ```

* after:
    ```
    {"count":1,"data":[{"a":{"i":2},"k":1}]}
    ```

Reply:
Based on offline conversation, reverting v1 to previous behavior.